definitions.d.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * [Color variant](/foundation/colors#component-colors)
+ */
+declare type Color = 'primary' | 'secondary' | 'selected' | 'success' | 'warning' | 'danger' | 'help' | 'info' | 'note' | 'light' | 'dark';
```

Comment:
⚠️ Not all colors can be always used. There are multiple "dictionaries" that group colors suitable for certain use case, see [RUI docs](https://react-ui.io/foundation/colors#applying-colors).

Does it make sense to separate them in TS? (I think it does, but maybe not now.)

Reply:
> Does it make sense to separate them in TS?

Yes, it makes sense to do it now.